packages/webapp/pages/account/notifications.tsx

```diff
@@ -150,22 +150,13 @@ const AccountNotificationsPage = (): ReactElement => {
       NotificationCategory.ReadingReminder,
     );
 
-    if (value) {
-      trackEvent({
-        event_name: AnalyticsEvent.ScheduleReadingReminder,
-        extra: JSON.stringify({
-          hour: readingTimeIndex,
-          timezone: user?.timezone,
-        }),
-      });
-      subscribePersonalizedDigest({
-        type: UserPersonalizedDigestType.ReadingReminder,
-      });
-    } else {
-      unsubscribePersonalizedDigest({
-        type: UserPersonalizedDigestType.ReadingReminder,
-      });
```

Comment:
I think we probably shouldn't have disabled these, as it's now impossible to subscribe to reading reminders (no mutations are triggered)